src/web/lib/contributions.ts

```diff
@@ -149,3 +150,20 @@ export const getArticleCountConsent = (): Promise<boolean> => {
         });
     });
 };
+
+export const noBannerUntilLater = (): boolean => {
```

Comment:
I think the approach is a good one. I think I'd call this function something more specific like `withinLocalCachePeriod` to make it really clear what it relates to.

Reply:
I've renamed to:
`withinLocalNoBannerCachePeriod`
`setLocalNoBannerCachePeriod`